classes/Tutor.php

```diff
@@ -837,13 +856,134 @@ public static function create_database() {
 			PRIMARY KEY (withdraw_id)
 		) $charset_collate;";
 
+		$orders_table = "CREATE TABLE {$wpdb->prefix}tutor_orders (
+			id BIGINT(20) UNSIGNED AUTO_INCREMENT,
+			payment_id BIGINT(20) UNSIGNED COMMENT 'Unique payment id from payment gateway',
+			user_id BIGINT(20) UNSIGNED NOT NULL,
+			status ENUM('pending', 'completed', 'cancelled', 'refunded', 'partially-refunded') NOT NULL,
+			sub_total_price DECIMAL(13, 2) NOT NULL, -- price calculation based on course sale price
+			total_price DECIMAL(13, 2) NOT NULL,
+			order_price DECIMAL(13, 2) NOT NULL,
+			coupon_code VARCHAR(255),
+			discount_type ENUM('percentage', 'flat'),
+			discount_amount DECIMAL(13, 2),
+			discount_reason TEXT,
+			tax DECIMAL(13, 2) COMMENT 'Tax percentage',
+			tax_amount DECIMAL(13, 2),
+			fees DECIMAL(13, 2),
+			earnings DECIMAL(13, 2),
+			payment_method VARCHAR(255),
+			payment_payloads LONGTEXT,
+			note TEXT,
+			created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
- We should not use `CURRENT_TIMESTAMP` to these field `created_at` and `update_at`. Because `CURRENT_TIMESTAMP` use mysql server time where we need user site time.
- `created_at_gmt` and `updated_at_gmt` needed to store 0 offset GMT time.
- We should avoid use of `ENUM`